### PR TITLE
DI-2663: Airflow2 compliance

### PR DESCRIPTION
**Pull request by Rcharriol**
* [x] Updated [CHANGELOG file](https://github.com/jampp/wolfria/blob/develop/CHANGELOG.rst)
* [x] Updated [VERSION file](https://github.com/jampp/wolfria/blob/develop/VERSION.rst)
* [x] Updated versions on `docker-compose.yaml` image names.

### CODE REVIEW

File: airflow/init_airflow.sh [hunk 1]
```diff
@@ -11,26 +12,14 @@ create_connections() {
...
-airflow variables -i "${CONFIG_FILES_PATH}/variables.json"
-airflow pool -i "${CONFIG_FILES_PATH}/pools.json"
-create_connections
+
+airflow db init
+airflow variables import "${CONFIG_FILES_PATH}/variables.json"
+airflow pools import "${CONFIG_FILES_PATH}/pools.json"
+delete_connections
+airflow connections import "${CONFIG_FILES_PATH}/extra_connections.json"
+airflow connections import "${CONFIG_FILES_PATH}/default_connections.json"
```
Comment (NMonnerSans) [inline airflow/init_airflow.sh:25]: Are these equivalent to the old create_connections function?

**Review by NMonnerSans — APPROVED** [review]
SNMNM - VDM

**Review by viniolivieri — APPROVED** [review]

**Comment by viniolivieri** [comment]
CHANGELOG?

File: airflow/init_airflow.sh (hunk 1 above)
Comment (Rcharriol) [reply airflow/init_airflow.sh:25]: Yes, in that order they are.

i learned that the hard way

**Review by gonzalezzfelipe — APPROVED** [review]

**Pull request merged**